CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.6.1
+
+* Eliminate installation of unnecessary header files when building nupic.bindings. Install Version.hpp unconditionally for the sake of the deployment usage in .travis.yaml
+* Guard against empty array case in ArrayBase::_templatedStreamBuffer
```

Comment:
The "Guard..." comment is part to the core work on the "Fixed servicing of delayed links..." below